normandy/selfrepair/static/js/self_repair_runner.js

```diff
@@ -1,3 +1,8 @@
+import 'babel-polyfill'
+import xhr from './xhr.js'
+import Mozilla from './uitour.js'
+import Normandy from './normandy_driver.js'
```

Comment:
Oh! When I ran it normally, I used `https://normandy/` (my normal test domain). When I tested it with docker-compose, I used `https://localhost:8000`, which doesn't share local storage. This module only needs `uuid` once, and then it caches that value in localstorage, which is per domain. That explains it.